feat(content): New Alien Faction: The Gegno (#6610)
endless-sky/endless-sky: data/gegno/gegno outfits.txt

outfit "R02 Battlezone Battery"
	category Power
	licenses
		"Gegno Civilian"
	plural "R02 Battlezone Batteries"
	cost 32000
	mass 23
	thumbnail outfit/gbatterysmall
	"energy capacity" 2237
	"outfit space" -23
	description `Battlezone batteries are primarily used on the Gegno's medium-range combat vessels, holding a moderate amount of power for long lasting engagements. They also fit nicely on larger vessels as a buffer to power their larger systems.`

outfit "R03 Warforge Battery"
	category Power
	licenses
		"Gegno Civilian"
	plural "R03 Warforge Batteries"
	cost 61700
	mass 41
	thumbnail outfit/gbatterymed
	"energy capacity" 4469
	"outfit space" -41
	description `Warforge batteries have proved their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage.`

outfit "R04 Crusade Battery"
	category Power
	licenses
		"Gegno Civilian"
		"Vi Evocati"
		"Vi Centurion"
		"Vi Lord"
	plural "R04 Crusade Batteries"
	cost 136000
	mass 83
	thumbnail outfit/gbatterylarge
	"energy capacity" 8885
	"outfit space" -83

> 	description `As the largest battery mounted on Gegno ships, the Crusade Battery holds a large amount of energy, more that it would take a small generator a decent amount of time to produce in order to support the energy hungry warships of both the Scin and the Vi. However, the Gegno are inefficient at increasing density with size, causing these batteries to fall behind similarly sized options.`

Reviewer: Okay that these batteries dont scale but mb they could have smth added to them when they get biggger? To do a bit of a niche? Or are these meant to kinda suck except for their cost? In that case maybe this bigger one could be cheaper per outfit space or smth.

Rn this one requires more licenses but doesnt bring anything on top of the other one.

> 	description `As the largest battery mounted on Gegno ships, the Crusade Battery holds a large amount of energy, more that it would take a small generator a decent amount of time to produce in order to support the energy hungry warships of both the Scin and the Vi. However, the Gegno are inefficient at increasing density with size, causing these batteries to fall behind similarly sized options.`

Author: It should be "more than", good catch.

outfit "C7 Brawl Cell"
	category Power
	licenses
		"Gegno Civilian"
	cost 68100
	mass 39
	thumbnail outfit/vicoresmall
	"energy generation" 2.5
	"heat generation" 3.9
	"outfit space" -39
	description `Unlike other C-class cores, C7 Brawl Cells are mounted horizontally. This design, however, still serves the Gegno well, though it will likely be phased out in favor of larger generators becoming more compact and efficient in the future.`

outfit "C17 Warzone Core"
	category Power
	licenses
		"Gegno Civilian"
	cost 2472000
	mass 87
	thumbnail outfit/vicoremed
	"energy generation" 14.1
	"heat generation" 38.3
	"outfit space" -87
	description `As did humans, the Gegno began developing nuclear technology once their cities grew massive enough to require it. The Warzone Core is a medium-sized nuclear fission reactor that is a suitable power source for most of the Gegno's large ships, though it looks even more discomforting than a human one.`

outfit "C27 Campaign Core"
	category Power
	licenses
		"Gegno Civilian"
		"Vi Evocati"
		"Vi Centurion"

> 		"Vi Lord"

Reviewer: these licenses feel a bit weird to me, why require all 4 or just the civilian? am I missing smth?
Are these meant to be different ways to prove yourself? I just don't see the added benefit of having these 3 be separate instead of being a license that would bring the same as all 3

Author: Yes it's a lore thing. Civilian means access to the basic tech and ships. Vi Evocati means access to basic Vi tech. Centurion, intermediary tech, Lord, strongest tech. They are a kind of "ranking" system. 

They come from these Wiki definitions, and are used within Vi society:

An evocatus (plural evocati) was a soldier in the Ancient [Roman army](https://en.wikipedia.org/wiki/Roman_army) who had served out his time and obtained an honorable discharge ([honesta missio](https://en.wikipedia.org/wiki/Honesta_missio)) but had voluntarily enlisted again at the invitation of the [consul](https://en.wikipedia.org/wiki/Roman_consul) or other commander.

A centurion ([/sɛnˈtjʊəriən/](https://en.wikipedia.org/wiki/Help:IPA/English); [Latin](https://en.wikipedia.org/wiki/Latin_language): centurio [[kɛn̪ˈt̪ʊrioː]](https://en.wikipedia.org/wiki/Help:IPA/Latin), pl. centuriones; [Greek](https://en.wikipedia.org/wiki/Greek_language): κεντυρίων, [translit.](https://en.wikipedia.org/wiki/Romanization_of_Greek) kentyríōn, or Greek: ἑκατόνταρχος, [translit.](https://en.wikipedia.org/wiki/Romanization_of_Greek) hekatóntarkhos) was a position in the [Roman army](https://en.wikipedia.org/wiki/Roman_army) during [classical antiquity](https://en.wikipedia.org/wiki/Classical_antiquity), nominally the commander of a [century](https://en.wikipedia.org/wiki/Centuria) ([Latin](https://en.wikipedia.org/wiki/Latin_language): centuria), a [military unit](https://en.wikipedia.org/wiki/Military_unit) of around 80 [legionaries](https://en.wikipedia.org/wiki/Legionaries).

Lord is an appellation for a person or [deity](https://en.wikipedia.org/wiki/Deity) who has [authority](https://en.wikipedia.org/wiki/Authority), control, or [power](https://en.wikipedia.org/wiki/Power_(social_and_political)) over others, acting as a master, chief, or ruler.[[](https://en.wikipedia.org/wiki/Lord#cite_note-1)

Reviewer: So I guess you'll change the outfit licenses? Caus currently they all require all 3 its a bit weird

Author: Like I said above, that's intentional. First, you have to prove yourself as a Gegno Civilian someday, before you can do anything with factions that are warring in their space. Once you've done that, you're required to go up the ranks on each respective faction for the given tech. Some requires some of the licenses, some requires all of them, etc.. It's all relative to the tech you're trying to get.

Author: If something includes both weapon tech and advanced technology tech, for example, both Scin licenses are required for that Scin tech. Scin ranks are more mix-match for unlocks, and Vi ones are more progressive.

Reviewer: Seems good
Mb all military outfits are together and so thats why I thought that all outfits had all 3 licenses
I feel like gegno could be a good place to start the game in honestly

		"Scin Adjutant"
	cost 162000
	mass 45
	thumbnail outfit/medscplasma
	"engine capacity" -45
	"outfit space" -45
	thrust 26.8
	"thrusting energy" 2.8
	"thrusting heat" 7.9
	"flare sprite" effect/scplasma/med
		"frame rate" 0.4
	"flare sound" "scplasma medium"
	description `SC-14 class Thrusters power the Scin's average gunboat or transport, giving their ships reliable speed. They can quickly propel any medium-sized ship forward with reasonable efficiency.`

outfit "SC-14 Plasma Steering"
	category Engines
	licenses
		"Gegno Civilian"
		"Scin Adjutant"
	cost 156400
	thumbnail outfit/medscplasmasteer
	"engine capacity" -34
	mass 34
	"outfit space" -34
	turn 690
	"turning energy" 1.5
	"turning heat" 4.78
	"steering flare sprite" effect/scplasma/med
		"frame rate" 0.4
	"steering flare sound" "scplasma medium"
	description `Larger than SC-12 thrusters, SC-14 steering thrusters are one of the most powerful steering systems in Gegno space, allowing the Scin to reposition even their largest ships many times faster than their Vi counterparts.`

outfit "SC-15 Plasma Thrusters"
	plural "SC-15 Plasma Thrusters"
	category Engines
	licenses
		"Gegno Civilian"
		"Scin Hoplologist"

> 		"Scin Architect"

Reviewer: Same logic as for the other one, except there are 2 here.

Author: Same as the above, lore with the Scin. One is more weaponry focused, the other more technical. 

Hoplologist: Someone who is an [expert](https://en.wiktionary.org/wiki/expert) in [weapons](https://en.wiktionary.org/wiki/weapon) and [armour](https://en.wiktionary.org/wiki/armour) (especially [military](https://en.wiktionary.org/wiki/military) uses or knowledge thereof).

Architect (computing): A person who designs hardware, software, or networking applications and services of a specified type for a business or other organization.

	cost 2312000

> 	mass -112

Reviewer: ```suggestion
	mass 112
```
I assume that wasnt intended

		"hit effect" "small explosion"
		lifetime 55
		reload 80
		velocity 9
		"hardpoint offset" 3
		inaccuracy 2
		"turret turn" 1.2
		"firing energy" 120
		"firing heat" 445
		"shield damage" 95
		"hull damage" 280
		"hit force" 180
	description `Mounted on a turret, a Chloeric Cannon is much more devastating without the need to keep a target in front to land shots. It can consistently put out damage to a target's hull, and with multiple turrets, any shieldless vessel is forced to retreat after a decent enough bombardment.`
	
effect "chloeric fire"
	sprite effect/chfire
		"frame rate" 25
		"no repeat"
	lifetime 7
	

outfit "Ballistic Cannon"
	category Guns
	cost 1025000
	mass 96
	thumbnail "outfit/ballistic cannon"
	"outfit space" -96
	"gun ports" -1
	"weapon capacity" -96
	licenses
		"Gegno Civilian"
		"Vi Evocati"
		"Vi Centurion"
		"Vi Lord"
	weapon
		sprite projectile/ballistic
		sound ballistic
		"fire effect" "ballistic fire"
		"hit effect" "ballistic hit"
		"hardpoint offset" -12
		lifetime 100
		reload 130
		"missile strength" 18
		"stream"
		velocity 8
		inaccuracy 3
		"firing energy" 252
		"firing heat" 566
		"shield damage" 145
		"hull damage" 776
		"hit force" 360

> 	description `The Gegno Vi Ballistic Cannon tops its sister cannons as the highest caliber. Each cannon round is as powerful as a small rocket and can inflict tremendous damage to a target's exposed hull. Due to the size and power of these cannons, their shells have a slow reload and travel speed. As such they are often dodgeable by faster or smaller craft, but if able to be equipped in numbers,they  leave a deadly field of rain in their sight.`

Reviewer 2: ```suggestion
	description `The Gegno Vi Ballistic Cannon tops its sister cannons as the highest caliber. Each cannon round is as powerful as a small rocket and can inflict tremendous damage to a target's exposed hull. Due to the size and power of these cannons, their shells have a slow reload and travel speed. As such, they can often be dodged by faster or smaller craft, but if able to be equipped in numbers, they leave a deadly field of rain in their sight.`
```
The last sentence doesn't feel right.

outfit "Ballistic Cannon Turret"
	category Turrets
	cost 1370000
	mass 120
	thumbnail "outfit/ballistic turret"
	"outfit space" -120
	"required crew" 3
	"turret mounts" -1
	"weapon capacity" -120
	licenses
		"Gegno Civilian"
		"Vi Evocati"
		"Vi Centurion"
		"Vi Lord"
	weapon
		sprite projectile/ballistic
		"hardpoint sprite" hardpoint/ballistic
		sound ballistic
		"fire effect" "ballistic fire"
		"hit effect" "ballistic hit"
		lifetime 100
		reload 110
		"missile strength" 18
		velocity 8
		"hardpoint offset" -1
		inaccuracy 3
		"turret turn" 0.6
		"firing energy" 247
		"firing heat" 537
		"shield damage" 145
		"hull damage" 776
		"hit force" 360

> 	description `As the largest caliber cannon in the Gegno Vi's arsenal, Ballistic Cannons require a lot of work to be fitted on even the largest of ships. A huge, self-sustaining base with its own vent ports is necessary for one of these cannons to even be fitted on a turret mount, provided there are crew to operate it. Combining destructive force with adjustable aim, Ballisic Cannon turrets are the deadliest weapons of Vi warships.`

Reviewer 2: ```suggestion
	description `As the largest caliber cannon in the Gegno Vi's arsenal, Ballistic Cannons require a lot of work to be fitted on even the largest of ships. A huge, self-sustaining base with its own vent ports is necessary for one of these cannons to even be fitted on a turret mount, provided there are crew to operate it. Combining destructive force with adjustable aim, Ballisic Cannon Turrets are the deadliest weapons of Vi warships.`
```

effect "ballistic fire"
	sprite effect/balfire
		"frame rate" 15
		"no repeat"
	lifetime 25

effect "ballistic hit"
	sprite effect/explosion/medium
		"frame rate" 15
		"no repeat"
	sound "explosion huge"
	lifetime 36
	"velocity scale" 0.4
	"random angle" 360
	"random velocity" 1
	"random spin" 5
	"random frame rate" 5
	

outfit "Astuit Battery"
	category Turrets
	licenses
		"Gegno Civilian"
		"Scin Adjutant"
	cost 764000
	mass 28
	thumbnail outfit/astuit
	"outfit space" -28
	"turret mounts" -1
	"weapon capacity" -28
	weapon
		sprite projectile/astuit
		"hardpoint sprite" hardpoint/astuit
		sound astuit
		"hit effect" "astuit hit"
		lifetime 44
		reload 6
		velocity 8
		"hardpoint offset" 8
		inaccuracy 5
		"turret turn" 2.2
		"firing energy" 13.2
		"firing heat" 34
		"shield damage" 8.4
		"hull damage" 8.8
	description "The Gegno Scin are the first Gegno to advance plasma technology to the point where it can be used in combat. Asuit Batteries work very similar to human Energy Blasters, firing small condensed plasma projectiles at high rates of fire. With 4 blasters on a turreted mount, it produces a constant stream of damage upon a target."

> 	description `	Although each shot does not hit very hard, consistent fire adds up - but so does energy usage.`

Reviewer 2: These two paragraphs need some work, I'm just not sure what to do; I'll have another look when I have more time.

		lifetime 1
		velocity 150
		reload 6
		"burst count" 3
		"burst reload" 1
		cluster
		inaccuracy 0.2
		"firing energy" 3.7
		"firing heat" 10
		"shield damage" 12.4
		"hull damage" 20.2
	description `The Guile Pulse Laser is a recent experimental advancement in Scin Plasma weapon technology. Rather than rapid firing plasma slugs, this weapon fires concentrated pulses of energy that only last for a brief moment. Each blast impacts the target ship almost instantaneously with a powerful hit, and a consistent amount of fire from Guile Lasers is extremely effective against slow moving targets.` 
	description `	Guile pulse lasers have noticeable drawbacks as they quickly consume energy and generate notable heat per shot. Unlike a beam-type weapon, they are incapable of consistent output, and have a fairly short range.`

outfit Excavator
	category Guns
	mass 42
	"thumbnail" "outfit/mouthparts"
	"gun ports" -1
	"outfit space" -42
	"weapon capacity" -42
	weapon
		sound drill
		"hit effect" drilling
		lifetime 1
		velocity 18
		inaccuracy 90
		"reload" 1
		"shield damage" 8
		"hull damage" 14
		"minable damage" 104
		"hit force" -4
		"firing energy" 4
		"firing heat" 100

> 	description `Excavator Drills are large drills designed the Gegno that rip materials out of asteroids in an efficient manner. These types of drill systems are only found on ships specifically designed around them as they require substantial existing infrastructure to be functional.`

Reviewer 2: ```suggestion
	description `Excavator Drills are large drills designed by the Gegno that rip materials out of asteroids in an efficient manner. These types of drill systems are only found on ships specifically designed around them as they require substantial existing infrastructure to be functional.`
```

outfit "C7 Brawl Cell"
	category "Power"
	licenses
		"Gegno Civilian"
	cost 68100
	mass 39
	thumbnail "outfit/vicoresmall"
	"energy generation" 2.5
	"heat generation" 3.9
	"outfit space" -39
	description `Unlike other C-class cores, C7 Brawl Cells are mounted horizontally. This design still serves the Gegno well, though it will likely be phased out in favor of larger generators as they become more compact and efficient.`

outfit "C17 Warzone Core"
	category "Power"
	licenses
		"Gegno Civilian"
	cost 2472000
	mass 87
	thumbnail "outfit/vicoremed"
	"energy generation" 14.1
	"heat generation" 38.3
	"outfit space" -87
	description `As did humans, the Gegno began developing nuclear technology once their cities grew massive enough to require it. The Warzone Core is a medium-sized nuclear fission reactor that is a suitable power source for most of the Gegno's large ships, though it looks even more discomforting than a human one.`

outfit "C27 Campaign Core"
	category "Power"
	licenses
		"Gegno Civilian"
		"Vi Evocati"
		"Vi Centurion"
		"Vi Lord"
	cost 8145000
	mass 145
	thumbnail "outfit/vicorelarge"
	"energy generation" 22.3
	"heat generation" 62.9
	"outfit space" -145

> 	description `In order to power the largest of vessels along with their power-guzzling systems and engines, the Gegno designed the Campaign Fusion Reactor from earlier prototypes during their colony ship era. It's still used to this day and is efficient enough to satisfy the requirements of the largest of Gegno warships, but just looking at it raises safety concerns.`

Reviewer 3: > Campaign Fusion Reactor

Since the name of the actual outfit is the Campaign Core, idk if it makes sense to capitalize "fusion reactor" here.

outfit "RG3 Torch Steering"
	category "Engines"
	licenses
		"Gegno Civilian"
		"Vi Evocati"
		"Vi Centurion"
		"Vi Lord"
	cost 102000
	mass 64
	thumbnail "outfit/large torch steer"
	"engine capacity" -64
	"outfit space" -64
	turn 701
	"turning energy" 1.6
	"turning heat" 4.2
	"steering flare sprite" "effect/ion torch/large"
		"frame rate" 2.3
	"steering flare sound" "ion torch large"
	description `The oldest steering system used in Gegno space, RG3 Torch thrusters can only fit on the largest Gegno vessels, and are rarely seen as a result.`

outfit "SC-1 Plasma Engines"
	plural "SC-1 Plasma Engines"
	category "Engines"
	licenses
		"Gegno Civilian"
		"Scin Hoplologist"
		"Scin Architect"
	cost 128000
	mass 24
	thumbnail "outfit/tinyscplasma"
	"engine capacity" -21
	"outfit space" -21
	thrust 5.1
	"thrusting energy" 0.54
	"thrusting heat" 1.2
	turn 160
	"turning energy" 0.31
	"turning heat" 0.86
	"flare sprite" "effect/scplasma/tiny"
		"frame rate" 0.6
	"flare sound" "scplasma tiny"

> 	description `In order to power the tiny Halite frame, the Gegno Scin developed this smaller engine system. Based on their larger plasma thrusters, these engines are much more efficient than the Vi's bigger, more sluggish Ion Engines, which would never be compact or light enough to run on a fighter.`

Reviewer 3: ```suggestion
	description `In order to power the tiny Halite frame, the Gegno Scin developed this smaller engine system. Based on their larger plasma thrusters, these engines are much more efficient than the Vi's bigger, more sluggish ion engines, which would never be compact or light enough to run on a fighter.`
```

	"engine capacity" -45
	"outfit space" -45
	thrust 26.8
	"thrusting energy" 2.8
	"thrusting heat" 7.9
	"flare sprite" "effect/scplasma/med"
		"frame rate" 0.4
	"flare sound" "scplasma medium"
	description `SC-14 class Thrusters power the Scin's average gunboat or transport, giving their ships reliable speed. They can quickly propel any medium-sized ship forward with reasonable efficiency.`

outfit "SC-14 Plasma Steering"
	category "Engines"
	licenses
		"Gegno Civilian"
		"Scin Adjutant"
	cost 156400
	thumbnail "outfit/medscplasmasteer"
	"engine capacity" -34
	mass 34
	"outfit space" -34
	turn 690
	"turning energy" 1.5
	"turning heat" 4.78
	"steering flare sprite" "effect/scplasma/med"
		"frame rate" 0.4
	"steering flare sound" "scplasma medium"
	description `Larger than SC-12 thrusters, SC-14 steering thrusters are one of the most powerful steering systems in Gegno space, allowing the Scin to reposition even their largest ships many times faster than their Vi counterparts.`

outfit "SC-15 Plasma Thrusters"
	plural "SC-15 Plasma Thrusters"
	category "Engines"
	licenses
		"Gegno Civilian"
		"Scin Hoplologist"
		"Scin Architect"
	cost 2312000
	mass 112

> 	thumbnail "outfit/largescplasma"

Reviewer 3: Seeing this a lot: image files should have spaces in their names if the name is multiple words.

		sound "irate"
		"fire effect" "irate fire"
		"hit effect" "tiny explosion"
		lifetime 15
		reload 20
		velocity 22
		"hardpoint offset" 1.5
		inaccuracy 2
		"firing energy" 10
		"firing heat" 95
		"shield damage" 6
		"hull damage" 16
		"hit force" 40
	description `Irate Cannons are a rudimentary cannon technology that fire explosive projectiles made out of condensed energy and metal, making them effective against armor. They fire decently fast, but one cannon alone is barely enough to output any sort of meaningful damage, which is why they are often found clustered together in numbers.`

outfit "Irate Cannon Turret"
	category "Turrets"
	cost 576000
	mass 28
	thumbnail "outfit/irate turret"
	"turret mounts" -1
	"required crew" 2
	"outfit space" -28
	"weapon capacity" -28
	licenses
		"Gegno Civilian" 
		"Vi Evocati"
	weapon
		sprite "projectile/irate"
		sound "irate"
		"fire effect" "irate fire"
		"hardpoint sprite" "hardpoint/irate"
		"hit effect" "tiny explosion"
		lifetime 15
		reload 12
		velocity 22
		"hardpoint offset" 12
		inaccuracy 2
		"turret turn" 1
		"firing energy" 13
		"firing heat" 95
		"shield damage" 6
		"hull damage" 16
		"hit force" 40

> 	description `While initially opted out as a weapon for Vi ships due to the production of larger calliber turreted cannons, the Irate Cannon Turret is used as a lighter option on Vi ships that do not have a sizeable weapon capacity. Despite the lack of raw firepower, the key feature of an Irate cannon's rate of fire is doubled with two of these mounted together on an assisting rotating platform.`

Reviewer 3: ```suggestion
	description `While initially opted out as a weapon for Vi ships due to the production of larger calliber turreted cannons, the Irate Cannon Turret is used as a lighter option on Vi ships that do not have a sizeable weapon capacity. Despite the lack of raw firepower, the key feature of an Irate Cannon's rate of fire is doubled with two of these mounted together on an assisting rotating platform.`
```

	"mass" 15
	"outfit space" -15
	"weapon capacity" -15
	"gun ports" -1
	weapon
		"sprite" "projectile/guile pulse laser"
			"frame rate" 6
		sound "guile pulse"
		"hit effect" "heavy laser impact"
		"inaccuracy" 0.2
		"velocity" 150
		"lifetime" 1
		"reload" 6
		"burst count" 3
		"burst reload" 1
		"firing energy" 3.7
		"firing heat" 10
		"shield damage" 12.4
		"hull damage" 20.2
		"cluster"
	description `The Guile Pulse Laser is a recent experimental advancement in Scin plasma weapon technology. Rather than rapid firing plasma slugs, this weapon fires concentrated pulses of energy that only last for a brief moment. Each blast impacts the target ship almost instantaneously with a powerful hit, and a consistent amount of fire from Guile Lasers is extremely effective against slow moving targets.` 
	description `	Guile Pulse Lasers have noticeable drawbacks as they quickly consume energy and generate notable heat per shot. Unlike a beam-type weapon, they are incapable of consistent output, and have a fairly short range.`

outfit "Excavator"
	category "Guns"
	cost 0
	"thumbnail" "outfit/mouthparts"
	"mass" 42
	"outfit space" -42
	"weapon capacity" -42

> 	"gun ports" -1

Author: Would it be best to give the Gypsum some type of attribute that would be negative and the positive part on the Excavator? Maybe even use "Excavator" as the attribute? 

I'm assuming this would still show up in the outfitter if you had one though, just not sellable? (Versus showing up as purchasable.) (Is there perhaps a way to hide an outfit from the outfitter completely? Or if you have one will it always show?)

> 	"gun ports" -1

Reviewer 3: ```
ship:
  "excavator port" 1
  "excavator lock" -1
weapon:
  "excavator port" -1
  "excavator lock" 1
```

		sound "acuit"
		"inaccuracy" 3
		"turret turn" 1.3
		"velocity" 18
		"lifetime" 60
		"reload" 56
		"burst reload" 10
		"burst count" 1
		"firing energy" 130
		"firing heat" 204
		"shield damage" 170
		"hull damage" 290
		"hit force" 40
	description `With a better grasp on plasma and energy weaponry, the Gegno Scin developed a larger blaster weapon to mount on the Feldspar Carrier as a counter to the Gegno Vi's huge cannon weaponry. The Acuit Artillery Cannon outranges most weapons in Gegno space, and has a rate of fire higher than most other weapons, making it a very efficient bombardment weapon from afar.`

effect "acuit hit"
	sprite "effect/acuit hit"
		"frame rate" 18
		"no repeat"
	"lifetime" 18
	"velocity scale" -0.004

outfit "Guile Pulse Laser"
	category "Guns"
	licenses
		"Gegno Civilian"
		"Scin Hoplologist"
	cost 109000
	"thumbnail" "outfit/guile pulse laser"
	"mass" 15
	"outfit space" -15
	"weapon capacity" -15
	"gun ports" -1
	weapon
		"sprite" "projectile/guile pulse laser"
			"frame rate" 6
		sound "guile pulse"
		"hit effect" "heavy laser impact"
		"inaccuracy" 0.2
		"velocity" 150
		"lifetime" 1
		"reload" 6
		"burst count" 3
		"burst reload" 1

> 		"firing energy" 3.7

Reviewer 3: An Energy Blaster consumes 29 energy/second while being 5 tons and dealing 53/33 shield/hull damage.
This consumes 37 energy/second at 15 tons and 124/202 shield/hull damage, so the energy usage really isn't as high as the description would suggest.

outfit "Excavator"
	category "Guns"
	cost 0
	"thumbnail" "outfit/mouthparts"
	"mass" 42
	"outfit space" -42
	"weapon capacity" -42
	"gun ports" -1
	weapon
		sound "drill"
		"hit effect" "drilling"
		"inaccuracy" 90
		"velocity" 18
		"lifetime" 1
		"reload" 1
		"firing energy" 4
		"firing heat" 100
		"shield damage" 8
		"hull damage" 14
		"minable damage" 104
		"hit force" -4
	description `Excavator Drills are large drills designed by the Gegno that rip materials out of asteroids in an efficient manner. These types of drill systems are only found on ships specifically designed around them as they require substantial existing infrastructure to be functional.`

effect "drilling"
	sprite "effect/smoke"
		"no repeat"
	sound "explosion tiny"
	"lifetime" 20
	"random angle" 360
	"random frame rate" 5
	"random spin" 2
	"random velocity" 0.4
	"velocity scale" -0.01

outfit "Savagery Bludgeon"
	category "Guns"
	cost 0
	"thumbnail" "outfit/unknown"
	"mass" 10
	"outfit space" -10
	"weapon capacity" -10
	"gun ports" -1
	"unplunderable" 1
	weapon
		"hit effect" "bludgeon impact"
		"inaccuracy" 360
		"velocity" 30
		"lifetime" 1
		"reload" 1
		"shield damage" 8
		"hull damage" 40

> 		"hit force" 140

Reviewer 3: I still think this should have a longer reload.

Reviewer 4: Is this a mace attached to a swing arm so it can be swung like a wrecking ball at an opponent? Or is this like the giant ramming protrusion from the from of a triream or other naval ship?

If it is a mace on a swing arm, then a reasonably fast reload seems appropriate.
If it is a ramming protrusion on the front, then I'd say the hull damage should be upped a bit but the reload made vastly longer. Long enough to need ships to disengage, turn, and take another run at their target for it to "reload."

> 	description `A Savagery Bludgeon is a mace-like weapon found jammed into the Gegno Vi's Dunite duelist ships. They are used as melee weapons for close-quarter rams and jousts during duels. However, they barely have any use of a weapon outside of this as they merely scratch larger ship's hulls and barely put a dent into anything shielded.`

Reviewer 3: ```suggestion
	description `A Savagery Bludgeon is a mace-like weapon found jammed into the Gegno Vi's Dunite duelist ships. They are used as melee weapons for close-quarter rams and jousts during duels. However, they barely have any use as a weapon outside of this as they merely scratch larger ship's hulls and barely put a dent into anything shielded.`
```

# Special Outfits

outfit "Gegno Civilian License"
	category "Special"
	thumbnail "outfit/unknown"
	description "You have been accepted as a titular Gegno citizen that provides benefits for the Gegno race, and have been awarded access to their civilian technology."

outfit "Vi Lord License"
	category "Special"
	thumbnail "outfit/unknown"
	description "A majority of the Vi's military hierarchy recognizes you as a respectable foe and invaluable asset for their ambitions. The responsibility of maintaining the largest of Vi vessels is now within your grasp."
	
outfit "Vi Centurion License"
	category "Special"
	thumbnail "outfit/unknown"
	description "Amongst the Vi's Evocati you have proven yourself as a powerful leader in combat and a successful user of judgment. Access to a portion of the Vi's military arsenal awaits your interests."

outfit "Vi Evocati License"
	category "Special"
	thumbnail "outfit/unknown"
	description "The Gegno Vi have recognized you as a committed warrior to their cause, and see usefulness in you. Low level Vi technology now becomes accessible for your leisure."

outfit "Scin Hoplologist License"
	category "Special"
	thumbnail "outfit/unknown"

> 	description "Your intelligence is most desirable in the eyes of the Gegno Scin, and your reward is the allowance of usage of advanced Scin weaponry."

Reviewer 3: ```suggestion
	description "Your intelligence is most desirable in the eyes of the Gegno Scin, and your reward is the usage of advanced Scin weaponry."
```

outfit "R02 Battlezone Battery"
	plural "R02 Battlezone Batteries"
	category "Power"
	licenses
		"Gegno Civilian"
	cost 32000
	thumbnail "outfit/battlezone battery"
	"mass" 23
	"outfit space" -23
	"energy capacity" 2237
	description `Battlezone Batteries are primarily used on the Gegno's medium-range combat vessels, holding a moderate amount of power for long-lasting engagements. They also fit nicely on larger vessels as a buffer to power their more energy hungry systems.`

outfit "R03 Warforge Battery"
	plural "R03 Warforge Batteries"
	category "Power"
	licenses
		"Gegno Civilian"
	cost 61700
	thumbnail "outfit/warforge battery"
	"mass" 41
	"outfit space" -41
	"energy capacity" 4469
	description `Warforge Batteries have proven their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage.`

outfit "R04 Crusade Battery"
	plural "R04 Crusade Batteries"
	category "Power"
	licenses
		"Gegno Civilian"
		"Vi Evocati"
		"Vi Centurion"

> 		"Vi Lord"

Author: Yes because these are derivative based on the old Gegno civilain generation ships.

> 		"Vi Lord"

Reviewer: oh so the added licenses are more of a lore thing than actual licenses ok

	cost 136000
	thumbnail "outfit/crusade battery"
	"mass" 83
	"outfit space" -83
	"energy capacity" 8885
	description `As the largest battery mounted on power-hungry Gegno ships, the Crusade Battery holds an immense amount of energy, more than a Scrap Cell could produce in almost two minutes. However, the Gegno are inefficient at increasing density with size, causing these batteries to fall behind similarly sized options.`

outfit "C3 Scrap Cell"
	category "Power"
	licenses
		"Gegno Civilian"
	cost 54000
	thumbnail "outfit/scrap cell"
	"mass" 25
	"outfit space" -25
	"energy generation" 1.4
	"heat generation" 2.2

> 	description `The Scrap Cell is a small radiothermal generator that produces a limited amount of energy. It is only used in tandem with the Gegno's smaller-sized vessels and internals.`

Reviewer 3: ```suggestion
	description `The Scrap Cell is a small radiothermal generator that produces a limited amount of energy. It is only used on the Gegno's smaller-sized vessels, or on tandem with their less energy hungry internals.`
```

outfit "C7 Brawl Cell"
	category "Power"
	licenses
		"Gegno Civilian"
	cost 68100
	thumbnail "outfit/brawl cell"
	"mass" 39
	"outfit space" -39
	"energy generation" 2.5
	"heat generation" 3.9

> 	description `When developing the C7 Brawl Cell, the Gegno decided to scale up the smaller C3 Scrap Cell. Unfortunately, the new design became so tall that the Gegno struggled to fit it into some of their ships, requiring them to turn the entire thing sideways. This design still serves the Gegno well, though it will likely be phased out in favor of newer generators as they become more compact and efficient.`

Reviewer 4: This is just an amusing piece of lore. I hope you build on it in the future. 🚀

outfit "SC-12 Plasma Thruster"
	category "Engines"
	licenses
		"Gegno Civilian"
		"Scin Adjutant"
	cost 98000
	thumbnail "outfit/small plasma thruster scin"
	"mass" 25
	"outfit space" -25
	"engine capacity" -25
	"thrust" 10.6
	"thrusting energy" 0.8
	"thrusting heat" 2.9
	"flare sprite" "effect/scin flare/small"
		"frame rate" 0.5
	"flare sound" "scplasma small"
	description `Rather than developing on the Ion Torch design of the Vi, the Scin began work on a plasma-based thruster. They are more compact, weigh less, and produce significantly more thrust. However, since Ion Torches expel some of their heat in their thrust, the more stable and powerful SC thrusters tend to run hotter.`

outfit "SC-12 Plasma Steering"
	category "Engines"
	licenses
		"Gegno Civilian"
		"Scin Adjutant"
	cost 95800
	thumbnail "outfit/small plasma steering scin"
	"mass" 19
	"outfit space" -19
	"engine capacity" -19
	"turn" 270
	"turning energy" 0.55
	"turning heat" 1.8
	"steering flare sprite" "effect/scin flare/small"
		"frame rate" 0.5
	"steering flare sound" "scplasma small"

> 	description `The new SC plasma engines allow Gegno Scin ships to outmaneuver their opposing brothers. What might take a Vi ship a long time takes only a moment's worth on a Scin ship.`

Reviewer 4: ```suggestion
	description `The new SC plasma engines allow Gegno Scin ships to outmanoeuvre their opposing brothers. What might take a Vi ship a long time takes only a moment on a Scin ship.`
```

Spelling, and removing an extra word to keep the sentence smoother.

outfit "SC-14 Plasma Thruster"
	category "Engines"
	licenses
		"Gegno Civilian"
		"Scin Adjutant"
	cost 162000

> 	thumbnail "outfit/medium plasma steering scin"

Reviewer 4: The outfit name says "Thruster" but the thumbnail says "steering." Is this correct?

Author: Good catches. Got mixed up.

	"mass" 45
	"outfit space" -45
	"engine capacity" -45
	"thrust" 26.8
	"thrusting energy" 2.8
	"thrusting heat" 7.9
	"flare sprite" "effect/scin flare/med"
		"frame rate" 0.4
	"flare sound" "scplasma medium"
	description `SC-14 class thrusters power the Scin's average gunboat or transport, giving their ships reliable speed. They can quickly propel any medium-sized ship forward with reasonable efficiency.`

outfit "SC-14 Plasma Steering"
	category "Engines"
	licenses
		"Gegno Civilian"
		"Scin Adjutant"
	cost 156400

> 	thumbnail "outfit/large plasma engines scin"

Reviewer 4: The outfit name says "Steering" but the thumbnail name says "engines." Is this correct?

Author: Good catches. Got mixed up.

	"mass" 34
	"outfit space" -34
	"engine capacity" -34
	"turn" 690
	"turning energy" 1.5
	"turning heat" 4.78
	"steering flare sprite" "effect/scin flare/med"
		"frame rate" 0.4
	"steering flare sound" "scplasma medium"
	description `Larger than SC-12 engines, SC-14-class steering is one of the most powerful steering systems in Gegno space, allowing the Scin to reposition even their largest ships many times faster than their Vi counterparts.`

outfit "SC-15 Plasma Thrusters"
	plural "SC-15 Plasma Thrusters"
	category "Engines"
	licenses
		"Gegno Civilian"
		"Scin Hoplologist"
		"Scin Architect"
	cost 2312000

> 	thumbnail "outfit/medium plasma steering scin"

Reviewer 4: The outfit name says "thruster" and the thumbnail here says "steering." Is this correct?

Author: Good catches. Got mixed up.

# Weapons:

outfit "Irate Cannon"
	category "Guns"
	licenses
		"Gegno Civilian" 
		"Vi Evocati"
	cost 475000
	thumbnail "outfit/irate cannon"
	"mass" 12
	"outfit space" -12
	"weapon capacity" -12
	"gun ports" -1
	weapon
		sprite "projectile/irate"
		sound "irate"
		"fire effect" "irate fire"
		"live effect" "irate smoke" 12
		"hit effect" "tiny explosion"
		"inaccuracy" 2
		"velocity" 22
		"lifetime" 15
		"reload" 20
		"firing energy" 10
		"firing heat" 95
		"shield damage" 6
		"hull damage" 16
		"hit force" 40
	description `Irate Cannons are a rudimentary cannon technology that fire explosive projectiles made out of condensed scrap metal, making them effective against armor. They fire decently fast, but one cannon alone is barely enough to output any sort of meaningful damage, which is why they are often found clustered together in numbers.`

> outfit "Irate Cannon Turret"

Author: I prefer to keep the Cannon part in. The name length is fine.

	
effect "choleric fire"
	sprite "effect/chfire"
		"frame rate" 25
		"no repeat"
	"lifetime" 7

effect "choleric smoke"
	sprite "effect/smoke"
		"scale" 0.1
		"no repeat"
		"frame rate" 18
	"lifetime" 16
	"random spin" 6
	"random angle" 360
	"random velocity" 0.5
	"velocity scale" 0.4
	

outfit "Ballistic Cannon"
	category "Guns"
	licenses
		"Gegno Civilian"
		"Vi Evocati"
		"Vi Centurion"
		"Vi Lord"
	cost 1025000
	thumbnail "outfit/ballistic cannon"
	"mass" 96
	"outfit space" -96
	"weapon capacity" -96
	"gun ports" -1
	weapon
		sprite "projectile/ballistic"
		sound "ballistic"
		"fire effect" "ballistic fire"
		"live effect" "ballistic smoke" 36
		"hit effect" "ballistic hit"
		"inaccuracy" 3
		"velocity" 8
		"lifetime" 100
		"reload" 130
		"firing energy" 252
		"firing heat" 566
		"shield damage" 145
		"hull damage" 776
		"hit force" 360
		"stream"
		"missile strength" 18

> 	description `The Gegno Vi Ballistic Cannon tops its sister cannons as the highest caliber weapon that the Gegno field. Each cannon round can inflict tremendous damage to a target's exposed hull. Due to the size and power of these cannons, their shells have a slow reload time and travel speed. As such, they can often be dodged by faster or smaller craft, but if able to be equipped in numbers, they can create a deadly wall of fire in their sight.`

Reviewer 4: ```suggestion
	description `The Gegno Vi Ballistic Cannon tops its sister cannons as the highest caliber weapon that the Gegno field. Each cannon round can inflict tremendous damage to a target's exposed hull. Due to the size and power of these cannons, their shells have a slow reload time and travel speed. As such, they can often be dodged by faster or smaller craft, but if able to be equipped in numbers, they can create a deadly wall of fire.`
```

Not entirely sure what the "in their sight." part adds to the sentence. The sentence reads quite well without it.

effect "ballistic fire"
	sprite "effect/balfire"
		"frame rate" 15
		"no repeat"
	"lifetime" 25

effect "ballistic hit"
	sprite "effect/explosion/medium"
		"frame rate" 15
		"no repeat"
	sound "explosion huge"
	"lifetime" 36
	"random angle" 360
	"random frame rate" 5
	"random velocity" 1
	"random spin" 5
	"velocity scale" 0.4
	
effect "ballistic smoke"
	sprite "effect/smoke"
		"scale" 0.1
		"no repeat"
		"frame rate" 10
	"lifetime" 30
	"random spin" 6
	"random angle" 360
	"random velocity" 0.5
	"velocity scale" 0.01

outfit "Astuit Battery"
	category "Turrets"
	licenses
		"Gegno Civilian"
		"Scin Adjutant"
	cost 764000
	thumbnail "outfit/astuit"
	"mass" 28
	"outfit space" -28
	"weapon capacity" -28
	"turret mounts" -1
	weapon
		sprite "projectile/astuit"
		"hardpoint sprite" "hardpoint/astuit"
		"hardpoint offset" 8
		sound "astuit"
		"hit effect" "astuit hit"
		"inaccuracy" 5
		"turret turn" 2.2
		"velocity" 8
		"lifetime" 44
		"reload" 6
		"firing energy" 13.2
		"firing heat" 34
		"shield damage" 8.8
		"hull damage" 9.4

> 	description "The Gegno Scin are the first Gegno to advance plasma technology to the point where it can be used in combat, although the energy requirement of such technology is still significant. Astuit Batteries work very similar to human Energy Blasters, firing small condensed plasma projectiles at high rates of fire. With four blasters on a turreted mount, it produces a constant stream of damage which adds up over time, despite the minor damage that each individual projectile creates."

Reviewer 4: Is there an individual "Astuit blaster" ? This description talks about Astuit batteries repeatedly, then after talking about energy blasters, is suddenly talking about putting four blasters on a turret, which makes it sound like it shifted focus to putting four energy blasters on the turret. A little confusing.

If there is an "astuit blaster", then this should be reworded a bit so that it doesn't sound like it is talking about energy blasters. If there isn't, then I'd suggested changing the term to "astuit cannon" to avoid confusion with energy blasters. Alternatively, just use their full name. This is what I have in the suggestion box:

```suggestion
	description "The Gegno Scin are the first Gegno to advance plasma technology to the point where it can be used in combat, although the energy requirement of such technology is still significant. Astuit Batteries work very similar to human Energy Blasters, firing small condensed plasma projectiles at high rates of fire. With four Astuit Blasters on a turreted mount, it produces a constant stream of damage which adds up over time, despite the minor damage that each individual projectile creates."
```

Author: There currently is not Astuit blaster, but there might be in the future? Not sure really. But yes this looks better.

effect "astuit hit"
	sprite "effect/ast hit"
		"no repeat"
		"frame rate" 30
	"lifetime" 6
	"velocity scale" .001

> outfit "Acuit Artillery"

Reviewer 4: So there's both Astuit weaponry and Acuit weaponry? I could see these names getting confused fairly easily. If there's a reason for having them so similar, great. If not, might be worth considering making the names at least a little bit more different.

Author: They're both words used to imply a means of intelligence (synonyms of), which is the Scin's forte. It's similar usage to how Vi weaponry is named after heated-temper/attitudes (ballistic, irate...). Both the same naming convention. I've tried other options, but "Smart Laser" or "Intelligence Blaster" don't work well.

	plural "Acuit Artillery"
	category "Turrets"
	licenses
		"Gegno Civilian"
		"Scin Hoplologist"
	cost 1406000
	thumbnail "outfit/acuit"
	"mass" 81
	"outfit space" -81
	"weapon capacity" -81
	"turret mounts" -1
	"required crew" 1
	weapon
		sprite "projectile/acuit"
			"random start frame"
		"hardpoint sprite" "hardpoint/acuit"
		"hardpoint offset" 14
		"hit effect" "acuit hit"
		sound "acuit"
		"inaccuracy" 3
		"turret turn" 1.3
		"velocity" 18
		"lifetime" 60
		"reload" 56
		"burst reload" 10
		"burst count" 1
		"firing energy" 130
		"firing heat" 204
		"shield damage" 170
		"hull damage" 290
		"hit force" 40

> 	description `With a better grasp on plasma and energy weaponry, the Gegno Scin developed a larger blaster weapon to mount on the Feldspar Carrier as a counter to the Gegno Vi's huge cannon weaponry. Acuit Artillery outranges most weapons in Gegno space, while having a higher rate of fire as well, making them very efficient bombardment weapons.`

Author: Is this correct? "Acuit Artillery" is the name of the weapon, not necessarily referring to "artillery" in a broad sense.

> 	description `With a better grasp on plasma and energy weaponry, the Gegno Scin developed a larger blaster weapon to mount on the Feldspar Carrier as a counter to the Gegno Vi's huge cannon weaponry. Acuit Artillery outranges most weapons in Gegno space, while having a higher rate of fire as well, making them very efficient bombardment weapons.`

Author: Perhaps it should be "Acuit Artillery turrets outrange..."?

> 	description `With a better grasp on plasma and energy weaponry, the Gegno Scin developed a larger blaster weapon to mount on the Feldspar Carrier as a counter to the Gegno Vi's huge cannon weaponry. Acuit Artillery outranges most weapons in Gegno space, while having a higher rate of fire as well, making them very efficient bombardment weapons.`

Reviewer 5: Yeah, that sounds better.

effect "acuit hit"
	sprite "effect/acuit hit"
		"frame rate" 18
		"no repeat"
	"lifetime" 18
	"velocity scale" -0.004

outfit "Guile Pulse Laser"
	category "Guns"
	licenses
		"Gegno Civilian"
		"Scin Hoplologist"
	cost 109000
	"thumbnail" "outfit/guile pulse laser"
	"mass" 15
	"outfit space" -15
	"weapon capacity" -15
	"gun ports" -1
	weapon
		"sprite" "projectile/guile pulse laser"
			"frame rate" 6
		sound "guile pulse"
		"hit effect" "heavy laser impact"
		"inaccuracy" 0.2
		"velocity" 150
		"lifetime" 1
		"reload" 6
		"burst count" 3
		"burst reload" 1
		"firing energy" 3.7
		"firing heat" 10
		"shield damage" 12.4
		"hull damage" 20.2
		"cluster"

> 	description `The Guile Pulse Laser is a recent experimental advancement in Scin plasma weapon technology. Rather than rapid firing plasma slugs, this weapon fires concentrated pulses of energy that only last for a brief moment. Each blast impacts the target ship almost instantaneously with a powerful hit, and a consistent amount of fire from Guile Lasers is extremely effective against slow moving targets.` 

Reviewer 4: To be honest, this sounds like what Heavy Lasers should be. Also reminds me of the "Thunderhead lance" from EVN, which is a good thing. Thunderhead lances were beams the way beams should be used in human-tech-level areas. Very powerful, but with significant downsides, particularly range. They were daggers in a gunfight, but daggers that were *highly* lethal if they could get close enough to be used.

Author: Ironically but unintentionally, Halites also *look* like Thunderheads, so I find it fun you draw that similarity as well. I suppose the Heavy laser issue is due to the name not including beam. These are really pulse lasers, and heavy lasers are beams, which separate them.

Reviewer 4: That makes sense. The more I play in my "full playthrough," the less I like the lasers in human space. Too good as a regular "every ship can make good use of them" kind of weapon. Whereas the thunderhead was a "great if you can get into close quarters, but a poor choice for most ships" weapon.

		"hit effect" "bludgeon impact"
		"inaccuracy" 360
		"velocity" 30
		"lifetime" 1
		"reload" 1
		"shield damage" 8
		"hull damage" 40
		"hit force" 140
	description `A Savagery Bludgeon is a mace-like weapon found jammed into the Gegno Vi's Dunite duelist ships. They are used as melee weapons for close-quarter rams and jousts during duels. However, they barely have any use as a weapon outside of this as they merely scratch larger ship's hulls and barely put a dent into anything shielded.`
	
effect "bludgeon impact"
	sound "bludgeon impact"

# Special Outfits

outfit "Gegno Civilian License"
	category "Licenses"
	thumbnail "outfit/unknown"
	description "You have been accepted as a titular Gegno citizen that provides benefits for the Gegno race, and have been awarded access to their civilian technology."

outfit "Vi Lord License"
	category "Licenses"
	thumbnail "outfit/unknown"
	description "A majority of the Vi's military hierarchy recognizes you as a respectable foe and invaluable asset for their ambitions. The responsibility of maintaining the largest of Vi vessels is now within your grasp."
	
outfit "Vi Centurion License"
	category "Licenses"
	thumbnail "outfit/unknown"

> 	description "Amongst the Vi's Evocati you have proven yourself as a powerful leader in combat and a successful user of judgment. Access to a portion of the Vi's military arsenal awaits your interests."

Reviewer 4: This has an... Interesting rythmn to it. Is judgment a thing in the story? or is this just saying that the player has good judgement? The way it reads makes it sound like judgement is a weapon or piece of equipment that the player has succesfully used.

I'm not suggesting anything because this could just be the way the Vi view judgement, and thus is culturally appropriate.

Author: Centurion is a step above Evocati, where Evocati is a sort of "foot soldier". Vi rankings are based on Roman army titles. So, in this case, a Centurion has been helpful to the Vi and had access to some of their tech. I very much would like the player, if they choose to actually participate in the war, be restricted and given access to Gegno tech during that period.

From Wikipidea:

An evocatus (plural evocati) was a soldier in the Ancient [Roman army] who had served out his time and obtained an honorable discharge [honesta missio]) but had voluntarily enlisted again at the invitation of the [consul] or other commander.

A centurion was a position in the [Roman army], nominally the commander of a [century], a [military unit] of around 80 [legionaries].

[Vi Lord] I did myself though. Also, the Scin's "licenses" are based on real-life field names.

Author: Commenting on the Mace (these are so close Git combined this suggestion lol), it is more of a protrusion yes. Derpy as stated above suggested a reload longer for it. I don't think the damage really needs to be upped though, at least not terribly much more- it's done as a beam right now. but a large knockback force balances it as the ship will be slung backwards after a hit.
